Makefile

```diff
@@ -40,6 +40,8 @@ ADDITIONAL_TAGS ?=
 
 KEPLER_IMG ?= $(IMG_BASE)/kepler:$(KEPLER_VERSION)
 
+# E2E_TEST_IMG defines the image:tag used for the e2e test image
+E2E_TEST_IMG ?=$(IMG_BASE)/kepler-operator-e2e:$(VERSION)
```

Comment:
I have created - https://quay.io/repository/sustainable_computing_io/kepler-operator-e2e 
@rootfs do we need make additional changes for the bots to be able to push to this repo?

Reply:
@sthaha Are we also planning to use the e2e image outside Prow CI? Regarding Prow CI usage, we don't need to push the e2e image to any repo. It will build the image and push it to its internal registry

Reply:
How is the image pushed?

Reply:
Just like how bundle and operator images are built and pushed. [Here](https://github.com/openshift/release/blob/2644055d79d1ae8172280197e1cb281cca9cd90e/ci-operator/config/vprashar2929/kepler-operator/vprashar2929-kepler-operator-package.yaml#L25) we define the list of images to build and in the test's [steps](https://github.com/openshift/release/blob/2644055d79d1ae8172280197e1cb281cca9cd90e/ci-operator/step-registry/vprashar2929/tests/kepler/vprashar2929-tests-kepler-ref.yaml#L3) we make use of the build image